pkg/sql/explain_bundle.go

```diff
@@ -431,14 +432,19 @@ func makeStmtEnvCollector(ctx context.Context, ie *InternalExecutor) stmtEnvColl
 	return stmtEnvCollector{ctx: ctx, ie: ie}
 }
 
+// explainBundleUserSessionsDataOverride is an InternalExecutorOverride which
+// overrides the users to the RootUser.
+var explainBundleUserSessionDataOverride = sessiondata.InternalExecutorOverride{
+	User: security.MakeSQLUsernameFromPreNormalizedString(security.RootUser)}
```

Comment:
Is it safe to use a root user override here? If not, what is a better alternative?